media-video/noad/noad-0.8.8.ebuild

```
@@ -0,0 +1,69 @@
+# Copyright 2021-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+DESCRIPTION="Mark commercial breaks in VDR recordings"
+HOMEPAGE="https://github.com/madmartin/noad"
+SRC_URI="https://github.com/madmartin/noad/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+ffmpeg imagemagick libmpeg2"
+REQUIRED_USE="|| ( ffmpeg libmpeg2 )"
+
+DEPEND="
+	libmpeg2? ( media-libs/libmpeg2:= )
+	ffmpeg? ( media-video/ffmpeg )
+	imagemagick? ( media-gfx/imagemagick )"
+RDEPEND="${DEPEND}"
+BDEPEND="virtual/pkgconfig"
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	econf \
+		$(usex imagemagick '--with-magick') \
+		$(usex ffmpeg '' '--without-ffmpeg') \
```

You can use `usev` here too:
```suggestion
		$(usev !ffmpeg --without-ffmpeg) \
```

And the line below too.

---

@ceamac Interesting proposition, did not think about that. I changed the ebuild accordingly.